<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NightNotePrefs</class>
 <widget class="QDialog" name="NightNotePrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Night Notes preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxAskIfdelete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>show warning dialog before deletion</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFolderSort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>show folders in the beggining of the list</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxWrapText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>wrap text by default</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabFolders">
    <attribute name="title">
     <string>Folders</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>4</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Working folder for Night Notes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPath">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NightNotePrefs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NightNotePrefs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
